### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by unityali99**
Completed unit tests for middleware functions

### CODE REVIEW

File: tests/units/checkTokenMiddleware.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    req.headers.token = process.env.INVALID_TOKEN;
+    await checkTokenMiddleware(req, res);
+    expect(res.status).toBeCalledWith(400);
+    expect(res.json).toBeCalledWith({
+      status: false,
+      message: "Invalid token",
+    });
+  });
+  it("Should respond with code 400 if there is no user related to the provided token ", async () => {
+    req.headers.token = process.env.TOKEN_NOT_EXIST;
```
Comment (ehsangazar) [inline tests/units/checkTokenMiddleware.test.js:18]: why process.env? just give it a sample data

File: tests/units/userAuthMiddleware.test.js [hunk 2]
```diff
@@ -13,24 +13,28 @@ describe("userAuthMiddleware", () => {
...
+  it("Should respond with code 400 if there is no user related to the token", async () => {
+    req.headers.auth = process.env.TOKEN_NOT_EXIST;
+    await userAuthMiddleware(req, res);
+    expect(res.status).toBeCalledWith(400);
+    expect(res.json).toBeCalledWith({
+      status: false,
+      message: "User does not exist",
+    });
+  });
   // it("Should respond with code 400 if there is no user related to the token", async () => {
```
Comment (ehsangazar) [inline tests/units/userAuthMiddleware.test.js]: remove the unneccessary comments please

Comment (unityali99) [reply tests/units/userAuthMiddleware.test.js:34]: Will fix it

File: tests/units/checkTokenMiddleware.test.js (hunk 1 above)
Comment (unityali99) [reply tests/units/checkTokenMiddleware.test.js:18]: The tokens may be needed in other unit and integration tests. By putting it in the .env, our code will be cleaner without redundancy.

**Pull request merged**